project/src/engines/hoomd/project.py

```diff
@@ -32,34 +32,60 @@ def run_hoomd(job):
     structure = ff.apply(filled_box)
 
     write_gsd(structure, job.fn("init.gsd"), ref_distance=rd, ref_energy=re)
-    # NOTE: create_hoomd3_forcefield depends on PR
-    # https://github.com/mosdef-hub/mbuild/pull/871
+    # ref_distance: angstrom -> nm
+    # ref_energy: kcal/mol -> kJ/mol
+    # ref_mass: dalton -> amu
     snapshot, forcefield, ref_vals = create_hoomd3_forcefield(
-        structure, ref_distance=10, ref_energy=1 / 4.184
+        structure, ref_distance=0.1, ref_energy=1 / 4.184, ref_mass=0.9999938574
     )
 
     device = hoomd.device.auto_select()
-    simulation = hoomd.Simulation(device=device, seed=job.sp.replica)
-    simulation.create_state_from_snapshot(snapshot)
+    sim = hoomd.Simulation(device=device, seed=job.sp.replica)
+    sim.create_state_from_snapshot(snapshot)
     gsd_writer = hoomd.write.GSD(
-        filename=job.fn("traj.gsd"),
+        filename=job.fn("trajectory.gsd"),
         trigger=hoomd.trigger.Periodic(10000),
         mode="ab",
     )
-    simulation.operations.writers.append(gsd_writer)
+    sim.operations.writers.append(gsd_writer)
+
+    logger = hoomd.logging.Logger(categories=["scalar"])
+    logger.add(sim, quantities=["timestep", "tps"])
+    thermo_props = hoomd.md.compute.ThermodynamicQuantities(
+        filter=hoomd.filter.All()
+    )
+    sim.operations.computes.append(thermo_props)
+    logger.add(
+        thermo_props,
+        quantities=[
+            "kinetic_energy",
+            "potential_energy",
+            "pressure",
+            "kinetic_temperature",
+            "volume",
```

Comment:
What about the bond, angle, etc... energies?

Reply:
I can add those, but they may not be comparable across engines.

Reply:
I guess the bond energies will be 0 since they are rigid (same with angles in water?). But it's still a good idea to have them available even if we end up not using that data.

Reply:
I agree.